src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -494,7 +496,7 @@ void receiveSimpleMessage(VertxTestContext context) throws InterruptedException,
                     long offset = jsonResponse.getLong("offset");
 
                     assertThat(kafkaTopic, is(topic));
-                    assertThat(value, is(sentBody));
+                    assertThat(value, is(sentBody + "-0"));
```

Comment:
who is adding this "-0" ? tbh who reads the test gets confused and I am the first. I read that `sentBody` is "Simple message" at the top but then you are comparing with the same with "-0" suffix. I think it's really confusing.

Reply:
Inside our vertex producer, we are adding this suffix. If we send more than 1 message it is better to check for instance `Simple message-50` that is the latest instead of the same `Simple message`.  I can rename that variable `sentBody` to `message`

Reply:
tbh whatever the name is I still don't think it's clear for who reads the test. From outside, no one knows that the basicKafkaClient is going to add a suffix. So I read the test expecting "Simple message" but you are testing with "Simple message-0". At this point the interface of the basicKafkaClient is not clear: I pass a `sentBody` parameter, asking for sending "Simple message" but the client actually sends "Simple message-0". It should be changed tbh.